HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -4,7 +4,7 @@ class Airflow
   # Constants
   InfilPressureExponent = 0.65
 
-  def self.apply(model, runner, weather, spaces, hpxml_header, hpxml_bldg, cfa, nbeds,
```

Comment:
@joseph-robertson FYI

Reply:
@shorowit Are you intentionally _burying_ this change in this "buried" duct PR? 😄